develop/getting-started/launching-the-game.md

```diff
@@ -2,8 +2,19 @@
 title: Launching the Game
 description: Learn how to utilize the various launch profiles to start and debug your mods in a live game environment.
 authors:
-  - IMB11
-  - Tenneb22
+  - amymialee
+  - andrew6rant
+  - draylar
+  - fracturedcode
+  - friendly-banana
+  - imb11
+  - jamieswhiteshirt
+  - mkpoli
```

Comment:
This was from moddedmc.wiki as well, this is original content not from the fabric wiki. Please double check the other pages and if they were actually from the fabric wiki